jme3-core/src/main/java/com/jme3/system/AppSettings.java

```diff
@@ -1350,4 +1351,65 @@ public boolean isUseRetinaFrameBuffer() {
     public void setUseRetinaFrameBuffer(boolean useRetinaFrameBuffer) {
         putBoolean("UseRetinaFrameBuffer", useRetinaFrameBuffer);
     }
+    
+    /**
+     * Get the Center Window state
+     *
+     * @return true for center the window in the middle of the screen
+     * @see #setCenterWindow(boolean)
+     */
+    public boolean getCenterWindow() {
+        return getBoolean("CenterWindow");
+    }
+    
+    /**
+     * True to enable center the window.
+     *
+     * @param center whether to center the window or not.
+     */
+    public void setCenterWindow(boolean center) {
+        putBoolean("CenterWindow", center);
+    }
+
+    
+    /**
+     * Get the position of the window's X Position
```

Comment:
The repetition of the word "position" makes this javadoc confusing.
Javadoc should mention that this is only valid for a non-full-screen window with CenterWindow set false.
Javadoc should mention that the X position is that of the upper-left corner of the content area, measured rightward from the left edge of the display.

Reply:
Javadoc should mention that the X position is that of the upper-left corner of the content area, measured rightward from the left edge of the display.